run_examples.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+#Licensed to the Apache Software Foundation (ASF) under one
+#or more contributor license agreements.  See the NOTICE file
+#distributed with this work for additional information
+#regarding copyright ownership.  The ASF licenses this file
+#to you under the Apache License, Version 2.0 (the
+#"License"); you may not use this file except in compliance
+#with the License.  You may obtain a copy of the License at
+#
+#http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing,
+#software distributed under the License is distributed on an
+#"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+#KIND, either express or implied.  See the License for the
+#specific language governing permissions and limitations
+#under the License.
+
+open -a Docker
+docker pull tinkerpop/gremlin-server
```

Comment:
In general, you can't mix server/driver versions so this might need to explicitly specify the version.

Reply:
I initially did specify the version, but was thinking that since this script would mainly be used before releases just to check that the examples are working, it might save some work to update the version each release if it only pulled the latest.  Do you think I should add it back?